source/configure/manage-user-surveys.rst

```diff
@@ -0,0 +1,81 @@
+
+1. Download the binary file from the GitHub repository.
+2. Go to **System Console > Plugin Management**.
+3. Next to **Upload Plugin**, select **Choose File**.
+4. Select the binary file, select **Open**, and then select **Upload**.
+
+Once the binary is uploaded, the **User Survey** integration is added in the navigation pane under **Plugins**.
+
+.. note::
+
+  Mattermost offers an additonal :doc:`User Satisfaction Surveys </manage/user-satisfaction-surveys>` option with limited customization options within the **Plugins** list where surveys are enabled by default. We recommend :ref:`disabling the User Satisfaction Surveys functionality <manage/user-satisfaction-surveys:how can surveys be disabled?>` when using this user survey integration.
+
+Upgrade
+~~~~~~~
+
+We recommend updating this integration as new versions are released. Generally, updates are seamless and don't interrupt the user experience in Mattermost.
+
+Visit the `Releases page <https://github.com/mattermost/mattermost-plugin-user-survey/releases>`_ for the latest release, available releases, and compatibiilty considerations.
+
+Enable
+------
+
+Go to **System Console > Plugins > User Survey** to enable this integration.
+
+Once the integration is installed and enabled, create surveys by completing configuration in the System Console, as described below.
+
+Create surveys
+--------------
+
+Under **Survey setup**, specify the date, time, and details for a new survey:
+
+1. **Send next survey at**: Specify the date and time when the survey will begin rolling out to users.
+
+  .. note::
+
+      A single survey can be active at a time. If you already have an active survey running, you'll need to reschedule your new survey to start on a date after the current survey expires. Alternatively, you can end the active survey early by selecting **End survey**.
+
+2. **Survey expiry**: Specify how long the survey will be open to responses in days.
+3. **Exclude specific teams**: Specify teams that shouldn't receive the survey. The survey is sent to all teams if left blank.
+4. **Survey message text** Customize the introductory message text users see when prompted to complete the survey.
+5. **Textual question**: (Optional) Specify a text-based question for the survey.
+6. Select **Save**. Your new survey is scheduled, and displays under **Active and past surveys** when the survey is shared with users.
+
+Active surveys
+--------------
+
+When a survey starts, all users recieve a direct message from a feedback bot containing the active survey. The survey includes one mandatory scale-based question, a mandatory text-based question, and 1 optional, customized text-based question. 
+
+Users must answer at least 1 question. When a user selects **Submit**, their responses are recorded.
+
+Export survey responses
+-----------------------
+
+Once a survey ends, select **Export responses** to download a CSV file containing NPS scores and user responses gatherered through the survey.
```

Comment:
Note that the report can be generated even while the survey is in progress. It will contain the data submitted so far.